### PR TITLE
[W4.4h][W13-2] Terrence Tan

### PR DESCRIPTION
**Pull request by TTTaus**
Added use case: Renaming of an existing tag

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -77,6 +77,42 @@ Use case ends.
...
 Use case resumes at step 2.
 
+=== Use case: Rename Tag
+
+*MSS*
+
+. User requests to rename tag
+. AddressBook returns and shows a list of unique tags
+. User requests to select a specific tag in the list
+. User requests to rename a specific tag in the list
```
Comment (okkhoy) [inline docs/DeveloperGuide.adoc]: 1. You can combine the above two. OR
2. You need to have a step where the `AddressBook` does something.
Remember, it was mentioned that the use case should show the interaction between the user and the system?

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -77,6 +77,42 @@ Use case ends.
...
+** 3a1. AddressBook shows an error message.
++
+Use case resumes at step 3.
+
+* 7a. The user cancels the confirmation.
++
+Use case resumes at step 6.
+
+* 9a. The tag rename operation was not successful.
++
```
Comment (okkhoy) [inline docs/DeveloperGuide.adoc]: In this case, you may want to let the user know that it was unsuccessful.

**Review by okkhoy — COMMENTED** [review]
You missed a bit on the format of the use case. Maybe review the topic on the module website, can?

**Comment by okkhoy** [comment]
@TTTaus 
Some comments added. Please take note and close the pull request.

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (TTTaus) [reply docs/DeveloperGuide.adoc:87]: Okay! What about this? 

. AddressBook returns and shows a list of unique tags
. User requests to select a specific tag in the list
. AddressBook confirms selection and prompts for an action
. User requests to rename a specific tag in the list

**Pull request closed without merging**